timeslotjoining.php

```diff
@@ -40,21 +43,30 @@
 
     switch ($action) {
         case 'join':
+            require_sesskey();
```

Comment:
I think this require_sesskey() should be before of the switch, I think it applies to all actions here, they are all writes